discord/http.py

```diff
@@ -660,12 +660,12 @@ def get_member(self, guild_id, member_id):
         return self.request(Route('GET', '/guilds/{guild_id}/members/{member_id}', guild_id=guild_id, member_id=member_id))
 
     def prune_members(self, guild_id, days, compute_prune_count, roles, *, reason=None):
-        params = {
+        payload = {
             'days': days,
             'compute_prune_count': 'true' if compute_prune_count else 'false',
-            'include_roles': roles
+            'include_roles': ', '.join(roles)
```

Comment:
Are we sure we can't just pass the array of snowflakes here as-is since it's JSON?

Reply:
I tried, there is no error returned, but at the same time, I am not sure if Discord is accepting it or not (since there is no indication whether it successfully pruned someone with the role)

Reply:
Might be worthwhile to test on the computing endpoint and seeing if the numbers change w/ role and w/o role